Libraries/LibJS/AST.h

```diff
@@ -670,9 +670,30 @@ class VariableDeclaration : public Declaration {
     NonnullRefPtrVector<VariableDeclarator> m_declarations;
 };
 
+class ObjectProperty final : public ASTNode {
+public:
+    ObjectProperty(NonnullRefPtr<Expression> key, NonnullRefPtr<Expression> value)
+        : m_key(key)
+        , m_value(value)
```

Comment:
We can `move()` here to avoid an unnecessary temporary copy of the pointers.
```suggestion
        : m_key(move(key))
        , m_value(move(value))
```